config/default/manager/manager.yaml

```diff
@@ -103,3 +103,71 @@ spec:
   selector:
     matchLabels:
       control-plane: mac-controller-manager
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: service
+  namespace: kubemacpool-system
+spec:
+  publishNotReadyAddresses: true
+  ports:
+    - port: 443
+      targetPort: 8000
+  selector:
+    kubemacpool-leader: "true"
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutator
```

Comment:
Add a comment talking about kustomize adding kubemacpool prefix and also tha caBundle is going to be updated by the kube-admission-webhook library.